module-code/app/securesocial/core/providers/ChatworkProvider.scala

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2015 Mikael Vallerie
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package securesocial.core.providers
+
+import play.api.libs.json.{ Reads, Json, JsValue }
+import play.api.{ Configuration, Environment }
+import play.api.libs.ws.{ WSAuthScheme, WSResponse, WSClient }
+import securesocial.core
+import securesocial.core._
+import securesocial.core.services.{ CacheService, HttpService, RoutesService }
+
+import scala.concurrent.{ ExecutionContext, Future }
+import ChatworkProvider.{ ErrorResponse, UserResponse }
```

Comment:
This import isn't used. There are many more.

Reply:
I remove redundant imports in the next commit.